code/src/components/Footer.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import styled from 'styled-components'
+import GlobalStyle from 'styles/GlobalStyle'
+
+const Footer = () => {
+  return (
+    <>
+      <GlobalStyle />
+      <FooterStyled>
+        <MadeBy>Created by Fiona Klacar | Technigo Web Development Bootcamp Spring 2023</MadeBy>
+        <IconsContainer>
+          <a href="https://www.linkedin.com/in/fiona-klacar/" target="_blank" rel="noopener noreferrer">
+            <Icon
+              src="/images/linkedin.svg"
+              alt="LinkedIn icon"
+              aria-label="view Fiona's linkedIn page" />
+          </a>
+          <a href="https://github.com/FionaKlacar" target="_blank" rel="noopener noreferrer">
+            <Icon
+              src="/images/github.svg"
+              alt="GitHub icon"
+              aria-label="view Fiona's GitHub page" />
+          </a>
+          <a href="https://fiona-klacar-portfolio.netlify.app/" target="_blank" rel="noopener noreferrer">
+            <Icon
+              src="/images/laptop.svg"
+              alt="link to Fiona's portfolio"
+              aria-label="view Fiona's portfolio" />
+          </a>
+        </IconsContainer>
+      </FooterStyled>
+    </>
```

Comment:
I've just worked with a footer all day long yesterday so really nice to see another way to deal with the icons and links and everything! Will remember this for my next project ☺️ 
Maybe you want to make the text smaller on mobile? Or the text on top of icons or other way around.

Reply:
That's great it has helped you! Thanks for pointing that out, I have fixed this now :)